src/os_push_notifications.eliomi

```diff
@@ -20,17 +20,17 @@
 
 (** Send push notifications to mobile clients.
 
-    This modules provides a simple OCaml interface to Google Cloud Messaging
+    This module provides a simple OCaml interface to Google Cloud Messaging
     (GCM) to send push notifications to mobile devices. It is recommended to use
     https://github.com/dannywillems/ocaml-cordova-plugin-push client-side to
-    receive push notification on the mobile.
+    receive push notifications on the mobile.
 
     This implementation is based on the payloads listed on this page:
     https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md
 
     Before using this module, you need to register your mobile application in
     GCM and save the server key GCM will give you. You need to pass this key to
-    the [send server_key notification options] function when you want to send a
+    [send server_key notification options] when you want to send a
```

Comment:
it's better than what there was before, but it seems you're pass something to `[send server_key notification options]` means that it's a function. Is it really the case?